File: web3rpc/rpc-specs/code-samples/javascript/kaia/configuration/gasPriceAt.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const blockNumber = 1;
  provider.kaia
    .gasPriceAt({ blockNumber }, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/kaia/configuration/gasPriceAt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

baseFee = "0x64"

w3 = Web3(Web3.HTTPProvider(host))
kaia_response = w3.kaia.gas_price_at(baseFee)

print(kaia_response)
